CMSgov/easi-app
# [NOREF] Introduce new Cypress E2E users

# NOREF

## Changes and Description

- Replace `TEST`, `ABCD`, `GRTB`, and a few other random users with consistent uses of `E2E1` or `E2E2`

This change was made to help provide a user that's used exclusively for Cypress/E2E tests, so that modifying LaunchDarkly user flags for users like `ABCD` or `TEST` (which are seeded in local data, and therefore good targets for local development) won't break our Cypress test suite.

## How to test this change

- Ensure the Cypress test suite runs in this PR as expected
- To test locally:
  - `scripts/dev up`
  - `yarn cypress run`

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
